Chatkit/Redux/Actions/ReceivedRemovedFromRoomAction.swift

```diff
@@ -0,0 +1,12 @@
+
+struct ReceivedRemovedFromRoomAction: Action {
+    
+    // MARK: - Properties
+    
+    let event: Wire.Event.RemovedFromRoom
```

Comment:
Do we ever foresee these concrete `Action`s having more than just an `event` property?  Only I'm wondering if we can get rid of all the boilerplate of these "wrapper" struct's and just make the event itself conform to `Action`?

`extension Wire.Event.RemovedFromRoom: Action {}`

Reply:
We might have other actions in future that will not have subscription events attached to them.